tests/smoke/test.cli.build.js

```diff
@@ -0,0 +1,32 @@
+/* @flow */
+import {spawn} from 'child_process';
+import {describe, it} from 'mocha';
+
+import {
+  webExt, addonPath, artifactsPath,
+} from './common';
+
+describe('web-ext build', () => {
+  it('webext build --source-dir SRCDIR --artifacts-dir OUTDIR', () => {
+    return new Promise((resolve, reject) => {
+      const webextProcess = spawn(webExt, [
+        'build',
+        '--source-dir', addonPath,
+        '--artifacts-dir', artifactsPath,
+      ]);
+
+      let errorData = '';
```

Comment:
this whole block of code looks like it can be wrapped up into a function that returns a promise, no?